testutils/src/main/java/com/amplifyframework/testutils/HubAccumulator.java

```diff
@@ -139,28 +144,20 @@ public HubAccumulator clear() {
      * block until the rest show up. If there are enough, they will
      * be returned immediately. The returned items will be cleared from the
      * accumulator.
-     * @param desiredQuantity Number of items being awaited
      * @return A list of desiredQuantity many items
```

Comment:
`desiredQuantity` is no longer a parameter

Reply:
Thanks, will fix